src/openrct2/ReplayManager.cpp

```diff
@@ -611,7 +624,7 @@ namespace OpenRCT2
             }
 
             if (!fs::is_regular_file(filePath))
-                return false;
+                throw std::invalid_argument(FormatStringID(STR_REPLAY_FILE_NOT_FOUND, file.c_str()));
```

Comment:
I'd expect a different message here:
1. Use fully resolved `filepath` instead of `file` here
2. say that `filepath` does not exist or is not a valid/readable file, instead of just "not found"